packages/string-store/src/lib/types/Array.ts

```diff
@@ -0,0 +1,40 @@
+import type { IType } from './IType';
+
+export function Array<SType, DType>(type: IType<SType, DType>, length: number | null = null) {
```

Comment:
```suggestion
export function Array<SType, DType>(type: IType<SType, DType>, length: number | null = null) {
```
It is largely unclear what SType and DType relate to here. Given that `Type` is implied by their usage, ignoring them, it's roughly as vague as `S` and `D`, which appear to have no real standardized use. Consider swapping those out.

Reply:
Bump, kyra